src/parser/parser.yy

```diff
@@ -121,6 +121,7 @@ class GraphScanner;
 %token KW_SNAPSHOT KW_SNAPSHOTS KW_LOOKUP
 %token KW_JOBS KW_JOB KW_RECOVER KW_FLUSH KW_COMPACT KW_SUBMIT
 %token KW_BIDIRECT
+%token KW_INSTALL KW_UNINSTALL KW_PLUGIN KW_SONAME KW_PLUGINS
```

Comment:
`.so`'s name is a key word ?

Reply:
so file name is not a key word.
There is a keyword called `sonam`.
This keyword is followed by the name of the so file.
e.g 
`INSTALL PLUGIN auth SONAME "ldap.so"`